transformation/src/test/test-files/inputs/AntibodyLot/part1.json

```diff
@@ -1,5 +0,0 @@
ody-characterizations/9597e2c6-532f-4e49-87f8-b5f020a0c23b/","/antibody-characterizations/a36bf222-a47c-4fb2-9f87-0f2891e38043/","/antibody-characterizations/630be6ee-475f-4cbc-afca-a31217516355/","/antibody-characterizations/b3552730-1bff-47d6-af39-e12f9d7d63a8/","/antibody-characterizations/e7f7a60d-3f5b-4f11-b91c-b9fb055668f3/","/antibody-characterizations/5bbe03b5-6b87-4339-b4e0-32d0341f68f5/","/antibody-characterizations/3c8bfcf3-91b4-444a-a3f1-e1dafc6d1022/","/antibody-characterizations/8d676f73-3d4c-4cb9-b1b0-f7164031eaae/"],"clonality":"polyclonal","date_created":"2013-09-13T13:23:43.409748-08:00","dbxrefs":["AR:AB_10000524","UCSC-ENCODE-cv:BHLHE40_(NB100-1800)"],"host_organism":"/organisms/rabbit/","lab":"/labs/michael-snyder/","lot_id":"A1","lot_id_alias":[],"lot_reviews":[{"biosample_term_id":"NTR:99999999","organisms":["/organisms/mouse/"],"biosample_term_name":"any cell type or tissue","targets":["/targets/BHLHE40-mouse/"],"status":"partially characterized","detail":"Primary characterization not reviewed."},{"biosample_term_id":"EFO:0001187","organisms":["/organisms/human/"],"biosample_term_name":"HepG2","targets":["/targets/BHLHE40-human/"],"status":"characterized to standards","detail":"Fully characterized."},{"biosample_term_id":"EFO:0002791","organisms":["/organisms/human/"],"biosample_term_name":"HeLa-S3","targets":["/targets/BHLHE40-human/"],"status":"characterized to standards","detail":"Fully characterized."},{"biosample_term_id":"EFO:0002784","organisms":["/organisms/human/"],"biosample_term_name":"GM12878","targets":["/targets/BHLHE40-human/"],"status":"characterized to standards","detail":"Fully characterized."},{"biosample_term_id":"NTR:99999999","organisms":["/organisms/human/"],"biosample_term_name":"any cell type or tissue","targets":["/targets/BHLHE40-human/"],"status":"partially characterized","detail":"Primary characterization not reviewed."},{"biosample_term_id":"EFO:0001196","organisms":["/organisms/human/"],"biosample_term_name":"IMR-90","targets":["/targets/BHLHE40-human/"],"status":"characterized to standards","detail":"Fully characterized."},{"biosample_term_id":"EFO:0002067","organisms":["/organisms/human/"],"biosample_term_name":"K562","targets":["/targets/BHLHE40-human/"],"status":"characterized to standards","detail":"Fully characterized."},{"biosample_term_id":"EFO:0001184","organisms":["/organisms/human/"],"biosample_term_name":"HEK293T","targets":["/targets/BHLHE40-human/"],"status":"characterized to standards","detail":"Fully characterized."}],"notes":"DEC-1 is a synonym for BHLHE40","product_id":"NB100-1800","purifications":["other"],"schema_version":"9","source":"/sources/novus/","status":"released","submitted_by":"/users/27e105ca-c741-4459-bf17-90e003508639/","targets":["/targets/BHLHE40-human/","/targets/BHLHE40-mouse/"],"title":"ENCAB000AEK","url":"http://www.novusbio.com/DEC1-Antibody-_NB100-1800.html?rr=true","used_by_biosample_characterizations":[],"uuid":"0cb167a6-52b5-4b2e-a626-715072090208"}
```

Comment:
This is an old file that I think was left here by mistake (it was replaced by `part-1.json`), so I removed it and the corresponding outputs.